core/vm/interpreter.go

```diff
@@ -112,17 +112,21 @@ func (in *Interpreter) Run(snapshot int, contract *Contract, input []byte) (ret
 		op    OpCode        // current opcode
 		mem   = NewMemory() // bound memory
 		stack = newstack()  // local stack
+		stackCopy = newstack()
```

Comment:
Does `stackCopy` really need to exist in this scope? Can't it just be created on the fly when needed?